mods/lord/Blocks/remains/src/common_functions.lua

```diff
@@ -0,0 +1,49 @@
+-- функция получения случайного лута по списку
+local function get_random_items(count, items, groups)
+	math.randomseed(os.time())
+	local result_items = {}
+	local i = 0
+	for _, row in pairs(items) do
+		if math.random(row.rarity) == 1 then
```

Comment:
забыл сказать:
для таких мест, где код будет выполнятся уже в процессе игры, а не при инициализации мода, особено если это критично для быстродействия, лучше добавлять локальные алиасы в начале файла:
```lua
local math_random, type, pairs
    = math.random, type, pairs
```
и использовать их:
```suggestion
		if math_random(row.rarity) == 1 then
```

FYI: тут это не критично (лутать скелет будут не особо то и часто), но при вынесении в `Core`, это точно нужно сделать, т.к. мы не знаем где и как будет использоваться эта ф-ция.